packages/main/src/components/FilterGroupItem/FilterGroupItem.jss.ts

```diff
@@ -0,0 +1,53 @@
+import { ThemingParameters } from '@ui5/webcomponents-react-base/lib/ThemingParameters';
+
+const styles = {
+  filterItem: {
+    width: '13rem',
+    height: 'fit-content',
```

Comment:
I think this is not working in our good old friend IE11. Is there any other CSS height possible?